crates/bevy_landmass/src/lib.rs

```diff
@@ -78,11 +80,24 @@ pub mod prelude {
   pub use crate::Velocity3d;
 }
 
-pub struct LandmassPlugin<CS: CoordinateSystem>(PhantomData<CS>);
+pub struct LandmassPlugin<CS: CoordinateSystem> {
+  _pd: PhantomData<CS>,
```

Comment:
Let's name this `marker` and make it last.

Reply:
https://github.com/andriyDev/landmass/pull/115/commits/22df66ef680b4f46b03a15ead1c373f1bbc810bf